jbmc/src/java_bytecode/java_bytecode_convert_method.cpp

```diff
@@ -3241,7 +3241,8 @@ void java_bytecode_convert_methodt::save_stack_entries(
     }
     if(replace)
     {
-      create_stack_tmp_var(tmp_var_prefix, tmp_var_type, block, stack_entry);
+      create_stack_tmp_var(
```

Comment:
No functional changes?

Reply:
Style commit I just pushed got rid of the change for some reason - fixed now.